src/metabase/audit_app/events/audit_log.clj

```diff
@@ -325,7 +325,16 @@
 
 (derive ::action-v2-event ::event)
 (derive :event/action-v2-execute ::action-v2-event)
+(derive :event/table-data-edit :event/action-v2-execute)
 
 (methodical/defmethod events/publish-event! ::action-v2-event
   [topic event]
-  (audit-log/record-event! topic event))
+  (let [table-data-edit-events #{:data-grid.row/create :data-grid.row/update :data-grid.row/delete :data-editing/undo :data-editing/redo}]
```

Comment:
would be nice if this can be represented as parent and child relation ship but upon checkin :data-editing/undo and :data-editing/redo does not derive from anything.